libp2p/NodeTable.cpp

```diff
@@ -580,6 +582,12 @@ void NodeTable::doHandleTimeouts()
     });
 }
 
+bool NodeTable::isAllowedAddress(bi::address const& _addressToCheck) const
```

Comment:
Maybe make it take `NodeIPEndpoint` parameter to avoid `.address()` duplication.